### PR TITLE
refactor: use branding icons for home tab, rewards pill

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Remove the custom ValoraV icon and use the icons from branding

### Test plan

Manually tested all affected screens

### Related issues

N/A

### Backwards compatibility

Yes

### Network scalability

N/A

### CODE REVIEW

File: src/navigator/TabNavigator.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default function TabNavigator({ route }: Props) {
           freezeOnBlur: false,
           lazy: false,
           tabBarLabel: t('bottomTabsNavigator.home.tabName') as string,
-          tabBarIcon: ValoraV,
+          tabBarIcon: Logo,
```
Comment (satish-ravi) [inline src/navigator/TabNavigator.tsx:61]: The drawer nav uses Home icon (it's only usage) which is essentially the same as Logo in the valora branding. It is a slightly different icon in the celo branding. I don't think we need it here and remove the Home icon when we remove the drawer, but I can update this to use Home if there's a strong reason to keep it separate.

File: scripts/sync_branding.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ echo $mobile_root
 cd "$mobile_root"
 
 # Please update the sha when valora branding updates are needed
-valora_branding_sha=19bc30b61b48a92c2a836ce1dafa3de25775e1b3
+valora_branding_sha=9724cbeb4cb1ddf883151afa588d09a74d400ddc
```
Comment (satish-ravi) [inline scripts/sync_branding.sh:24]: TODO(satish): update once https://github.com/valora-inc/valora-app-branding/pull/27 is merged

File: branding/celo/src/icons/Logo.tsx [hunk 3]
```diff
@@ -3,46 +3,28 @@ import { View, ViewStyle } from 'react-native'
...
-      break
-    default:
-      break
-  }
-
+export default function Logo({ size = 25, color, testID, style }: Props) {
   return (
     <View testID={testID} style={style}>
-      <Svg width={height} height={height} viewBox="0 0 25 25" fill="none">
+      <Svg width={size} height={size} viewBox="0 0 25 25" fill="none">
```
Comment (MuckT) [inline branding/celo/src/icons/Logo.tsx:16]: This view box is different than the ValoraV. Is that ok?

Comment (satish-ravi) [reply branding/celo/src/icons/Logo.tsx:16]: yes that should be fine

**Review by MuckT — APPROVED** [review]

**Pull request merged**